packages/react-components/src/components/ChatMessage/ChatMessageContent.tsx

```diff
@@ -72,6 +74,16 @@ const MessageContentWithLiveAria = (props: MessageContentWithLiveAriaProps): JSX
 
 const MessageContentAsRichTextHTML = (props: ChatMessageContentProps): JSX.Element => {
   const liveAuthor = _formatString(props.strings.liveAuthorIntro, { author: `${props.message.senderDisplayName}` });
+
+  /* @conditional-compile-remove(teams-inline-images) */
+  useEffect(() => {
```

Comment:
Add a dependency array to this useEffect, since we don't wanna trigger it everytime when we re-render

https://codedamn.com/news/reactjs/useeffect-dependency#:~:text=component%20gets%20updated.-,useEffect%20Dependency%20array,-The%20useEffect%20manages